server/routes/createGoal/index.ts

```diff
@@ -17,7 +18,7 @@ export default (router: Router, services: Services) => {
 
   router.use('/plan/:prisonNumber/goals/create', checkUserHasEditAuthority())
   router.get('/plan/:prisonNumber/goals/create', [
-    checkPrisonerSummaryExistsInSession,
```

Comment:
This is the request handler that we used previously for the GET route for creating a goal. It checks whether the prisoner summary is in session, and if no redirects the user to PLP's Overview page

The request handler `checkPrisonerSummaryExistsInSession` is still in use in most of our other routes, its just that this route now needs to do something slightly different; hence the change to use the new request handler 👉